olive/engine/packaging/packaging_config.py

```diff
@@ -35,10 +36,15 @@ class AzureMLModelsPackagingConfig(CommonPackagingConfig):
     description: Optional[str] = None
 
 
+class OrtGenAIPackagingConfig(ZipfilePackagingConfig):
+    pass
+
+
 _type_to_config = {
     PackagingType.Zipfile: ZipfilePackagingConfig,
     PackagingType.AzureMLModels: AzureMLModelsPackagingConfig,
     PackagingType.AzureMLData: AzureMLDataPackagingConfig,
+    PackagingType.OrtGenAI: OrtGenAIPackagingConfig,
 }
```

Comment:
Do we need a separate packaging type?
PackagingType.AzureMLModels and  PackagingType.Zipfile should also be used for LLMs that use generate() from onnxruntime-genai repo.

Reply:
No we don't. Pushed up a bad commit.